SETUP/tests/jsTests/formatPreviewTests.js

```diff
@@ -0,0 +1,113 @@
+/* global QUnit analyse */
+
+QUnit.test("Format preview test", function( assert ) {
+    let inLineTags = "[ibfg]|sc";
+    let suppress = {};
```

Comment:
const?

Reply:
I think the paramters I chose for the anayse function may not be the best. Maybe taking the whole `style` into it would be better and allow for the optional math markup in another MR.